cmd/hailstone/main.go

```diff
@@ -17,26 +18,34 @@ import (
 	hmsl "github.com/davidh-cyberark/brimstone/pkg/hasmysecretleaked"
 	pam "github.com/davidh-cyberark/brimstone/pkg/privilegeaccessmanager"
 	"github.com/davidh-cyberark/brimstone/pkg/utils"
+
+	"github.com/caarlos0/env/v6"
 )
 
 var (
-	version         = "dev"
-	TLS_SKIP_VERIFY = false
-
-	DEBUG = false
+	version = "dev"
+	DEBUG   = false
 )
 
-func main() {
-	idtenanturl := flag.String("idtenanturl", "", "PAM config ID tenant URL")
-	pcloudurl := flag.String("pcloudurl", "", "PAM config Privilege Cloud URL")
-	pamuser := flag.String("pamuser", "", "PAM config PAM User")
-	pampass := flag.String("pampass", "", "PAM config PAM Pass")
-	safename := flag.String("safename", "", "PAM config PAM Safe Name")
+type config struct {
```

Comment:
Can the struct from brimstone be re-used?

Reply:
They do not hold the exactly the same info but share the CyberArk credentials.
I have merged the common part in a struct `BaseConfig`